src/modules/error/components/Failure.js

```diff
@@ -0,0 +1,36 @@
+// @flow
+
+import type { TFunction } from 'react-i18next'
+
+import React from 'react'
+import styled from 'styled-components'
+import { translate } from 'react-i18next'
+import Icon from 'react-native-vector-icons/FontAwesome'
+import { Text } from 'react-native'
+
+const ViewContainer = styled.View`
+flex: 1;
+align-items: center;
+margin-top: 10%;
+`
+
+type PropsType = {|
+  error: Error,
+  t: TFunction
+|}
+
+export class Failure extends React.Component<PropsType> {
+  render () {
+    const {error, t} = this.props
+
+    console.error(error)
```

Comment:
Should we log this error here?

Reply:
Is there a better place to log it? I think this is just the most straightforward.

Reply:
Console logs should be avoided in production mode: https://facebook.github.io/react-native/docs/performance#using-consolelog-statements

Also test would output the errors. Maybe there is a better place for logging. e.g. the place where the exception was thrown.